.circleci/config.yml

```diff
@@ -0,0 +1,75 @@
+version: 2
+
+# https://circleci.com/blog/circleci-hacks-reuse-yaml-in-your-circleci-config-with-yaml/
+defaults: &defaults
+  docker:
+    - image: docker.io/weaveworks/wksctl-build:go-1.14.1-431fabe9
+      environment:
+        GOPATH: /go/
+        SRCDIR: /src/github.com/weaveworks/wksctl
+        KUBECTL_URL: https://dl.k8s.io/v1.10.5/kubernetes-client-linux-amd64.tar.gz
+        KUBECTL_CHECKSUM: da9d557989a0b9671a610f21642052febb8f70c3cf144c98a8a4f7ecab6bafe2
+  working_directory: /src/github.com/weaveworks/wksctl
+
+workflows:
+  version: 2
+  test-build-deploy:
+    jobs:
+      - build:
+          filters:
+            tags:
+              only: /.*/
+      - unit-tests: {}
+      - release:
+          requires:
+            - build
+            - unit-tests
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /.*/
+
+jobs:
+  build:
+    <<: *defaults
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run: make
+      - run:
+          name: Check that generated files have not been changed since checkout
+          command: |
+            echo "The following files have been modified since checkout or are unknown to Git:"
+            ! (git status --porcelain | grep -E '^( M)|(??)')
+      - run:
+          # Allow builds from forks
+          name: Push image if docker login is available
+          command: |
+            if [ -n "$DOCKER_IO_PASSWORD" ]; then
+              docker login -u "$DOCKER_IO_USER" -p "$DOCKER_IO_PASSWORD" docker.io
+              make push
```

Comment:
There is no `push` target in the Makefile.

(there is a `docker-push`, but it doesn't seem to do what we want)

Reply:
K, thanks.